packages/webapp/src/_app/ui/action-button.tsx

```diff
@@ -56,22 +60,37 @@ export const ActionButton = ({
     const cursorClass = disabled ? "cursor-not-allowed" : "cursor-pointer";
     const buttonClass = `${baseClass} ${size} ${widthClass} ${colorClass} ${cursorClass} ${className}`;
 
-    if (href) {
+    let targetProps = {
+        target,
+        rel: isExternal ? "noopener noreferrer" : undefined,
+    };
+
+    if (onClick) {
+        return (
+            <button
+                onClick={onClick}
+                type={isSubmit ? "submit" : "button"}
+                className={buttonClass}
+                disabled={disabled}
+            >
+                {children}
+            </button>
+        );
+    }
+
+    if (isExternal) {
         return (
-            <NextLink href={href}>
-                <a className={buttonClass}>{children}</a>
-            </NextLink>
+            <a className={buttonClass} href={href} {...targetProps}>
+                {children}
+            </a>
         );
     }
 
     return (
-        <button
-            onClick={onClick}
-            type={isSubmit ? "submit" : "button"}
-            className={buttonClass}
-            disabled={disabled}
-        >
-            {children}
-        </button>
+        <NextLink href={href}>
+            <a className={buttonClass} {...targetProps}>
+                {children}
+            </a>
+        </NextLink>
```

Comment:
I think this change will break submit buttons (fill all the fields and press enter). Also I don't think it's a good idea to replace the button semantics with a link inside forms.

Reply:
ohh nevermind... I just saw the button being returned above!

Reply:
nevermind again lol... the button will be reeturned only if onclick is present, but submit buttons don't always have it: https://github.com/eoscommunity/Eden/blob/main/packages/webapp/src/inductions/components/induction-profile-form.tsx#L245

Reply:
yeah, good catch!